VideoStreaming/SubViews/Views/VideoView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  VideoView.swift
+//  VideoStreaming
+//
+//  Created by Julian Llorensi on 17/09/2019.
+//  Copyright © 2019 ESPN. All rights reserved.
+//
+
+import UIKit
+
+class VideoView: UIView {
+    @IBOutlet weak var videoImageView: UIImageView!
+    @IBOutlet weak var videoTitle: UILabel!
+    @IBOutlet weak var videoDurationAndDate: UILabel!
+    
+    init(frame: CGRect, video: Video) {
+        super.init(frame: frame)
+        setViewInfoWith(video)
+        setUpImageView()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func resetViewInfoWith(_ video: Video) {
+        videoImageView.image = UIImage(named: video.imageName)
```

Comment:
If you see, these two methods are the same (DRY: Don`t Repeat Yourself principle). I think having only one public method is enough. Also, the `With` at the end of method name looks no swifty, ;). You can take a look at this https://swift.org/documentation/api-design-guidelines/ .
I would prefer something like `configure(with video: Video)`

Reply:
Done